src/html.sortable.js

```diff
@@ -554,6 +559,11 @@ var sortable = function (sortableElements, options) {
       })
     }
 
+    // max items
+    if (options.maxItems && typeof options.maxItems === 'number') {
```

Comment:
Sure, I don't know if a type check is even necessary, just sort of emulating what I saw elsewhere. The type check probably isn't necessary at all, it will act weird if people give a weird parameter but that's on them.